<launch>
  <group ns="ns1">
    <node  name="rslidar_node" pkg="rslidar_driver" type="rslidar_node" output="screen" >
      <!--param name="pcap" value="/home/tony-sti/lidar_data/lm75-170616-roadtest.pcap"/ -->
      <param name="device_ip" value="192.168.1.142" />
      <param name="port" value="2042"/>
      <param name="time_synchronization" value="true"/>
    </node>

    <node  name="cloud_node" pkg="rslidar_pointcloud" type="cloud_node" output="screen" >
      <param name="curves_path" value="$(find rslidar_pointcloud)/data/lidar1/curves.csv" />
      <param name="angle_path" value="$(find rslidar_pointcloud)/data/lidar1/angle.csv" />
      <param name="channel_path" value="$(find rslidar_pointcloud)/data/lidar1/ChannelNum.csv" />
    </node>
  </group>

  <group ns="ns2">
    <node  name="rslidar_node" pkg="rslidar_driver" type="rslidar_node" output="screen" >
      <param name="device_ip" value="192.168.1.149"/>
      <param name="port" value="2049"/>
      <param name="time_synchronization" value="true"/>
    </node>

    <node  name="cloud_node" pkg="rslidar_pointcloud" type="cloud_node" output="screen" >
      <param name="curves_path" value="$(find rslidar_pointcloud)/data/lidar2/curves.csv" />
      <param name="angle_path" value="$(find rslidar_pointcloud)/data/lidar2/angle.csv" />
      <param name="channel_path" value="$(find rslidar_pointcloud)/data/lidar2/ChannelNum.csv" />
    </node>
  </group>

  <node  name="multilidar_timesync" pkg="multilidar_timesync" type="timesync_node" output="screen" >
    <param name="scan1_topic" value="/ns1/rslidar_packets"/>
    <param name="scan2_topic" value="/ns2/rslidar_packets"/>

    <param name="skippackets1_topic" value="/ns1/skippackets_num"/>
    <param name="skippackets2_topic" value="/ns2/skippackets_num"/>
  </node>
</launch>
